files/qsub.pl

```diff
@@ -551,9 +548,22 @@ sub parse_script
     };
 
 
-    # Look for PBS directives for -o, -e, -N, -j
+    # Look for PBS directives for -o, -e, -N, -j if command they not given in the command line
     # If they are not set, add the commandline args from defaults
     # keys are slurm option names
+    my %orig_args = map { $_ => 1 } @orig_args;
```

Comment:
let's see if i get this right
```perl
my %orig_args = map { s/^-//; $_ => 1 } grep {m/^-.$/} @orig_args;  # only map the one-letter options, and remove the leading -
my @check_pbsopt = qw(j o N);
push (@check_opts, 'e') if $orig_args{j};
@check_pbsopt = grep {!$orig_args{$_}} @check_pbsopt;
```

Reply:
ok, and with arrayref `$orig_args`, you can't use `%orig_args` as name (otherwise `$orig_args` is not what you think it is)

Reply:
@stdweird `@orig_args` can be empty.